.flake8

```diff
@@ -9,4 +9,5 @@ exclude =
     doc/manual/source/conf.py
 max-line-length = 115
 # Ignore some style 'errors' produced while formatting by 'black'
-ignore = W503
+# E231, E271, and E272 are ignored for formatting purposes regarding space around ':' and docstrings
```

Comment:
please put all the details.  Which error is precisely for what and why we ignore it.